### ISSUE TITLE
FPM uses site-packages directory to store python packages

### ISSUE THREAD

**Issue by ObjatieGroba** [jordansissel/fpm#1781]
Python 3.9, debian 11, ARM64 

fpm -s python -t deb --python-package-name-prefix python3 --python-bin=python3 netifaces

Stores files in /usr/local/lib/python3.9/site-packages

Python path contains only  /usr/local/lib/python3.9/dist-packages

How can I fix it?

**Comment by jordansissel** [comment]
I agree fpm should make the best default it can. I haven’t checked the code
recently, but I recall FPM using Python’s own environment settings for
determining where the correct installation path is. Possible it’s guessing
the wrong place and maybe a bug fix will be easy? ;)

**Comment by amdei** [comment]
On my RaspberryPi with recent Debian 11 I got `site-packages` as python default:
```
# python3 -c 'from distutils.sysconfig import get_python_lib; print(get_python_lib())'
/usr/lib/python3.9/site-packages
```

May be `--install-lib` or even `--setup-py-arguments` fpm oprions could help?

**Comment by ObjatieGroba** [comment]
> On my RaspberryPi with recent Debian 11 I got site-packages as python default:

Same. sys.path is
`['', '/usr/lib/python39.zip', '/usr/lib/python3.9', '/usr/lib/python3.9/lib-dynload', '/usr/local/lib/python3.9/dist-packages', '/usr/lib/python3/dist-packages']`

**Comment by ObjatieGroba** [comment]
`get_python_lib` says 
`is_default_prefix = not prefix or os.path.normpath(prefix) in ('/usr', '/usr/local')`

if `is_default_prefix` it use `dist-packages` path. If not - `site-packages`. 
This means that fpm on debian generates not the same package as default setup.py installation.

In my opinion, fpm should use `dist-packages` as default force override on debian

**Comment by mkg20001** [comment]
This worked for me: `--python-install-lib /usr/lib/python3/dist-packages`
